R/app_server.R

```diff
@@ -11,6 +11,9 @@ app_server <- function(input, output, session) {
   # Collect user info.
   user <- reactiveValues()
   user$metrics_reweighted <- 0
+  credential_config <- get_golem_config("credentials", file = app_sys("db-config.yml"))
+  role_opts <- list(admin = as.list(credential_config $privileges$admin), 
+                    nonadmin = as.list(setdiff(credential_config$roles, credential_config $privileges$admin)))
```

Comment:
Whoa, is it kosher to have a space after `credential_config`?
```suggestion
  role_opts <- list(admin = as.list(credential_config$privileges$admin), 
                    nonadmin = as.list(setdiff(credential_config$roles, credential_config$privileges$admin)))
```

Reply:
Not sure how those got in there. They are now removed.